### PR TITLE
Nordrhein-Westfalen, Germany

### PR DESCRIPTION
**Pull request by sergiyprotsiv**
Here's the biggest state of Germany (>4M addresses), apparently they have just released their whole cadastre and more as open data: https://www.opengeodata.nrw.de/produkte/geobasis/lika/alkis_sek/.

The file I am using here is the direct download from https://www.opengeodata.nrw.de/produkte/geobasis/lika/alkis_sek/gebref/gebref_EPSG4647_ASCII.zip
However, that file requires merge over three fields to get to city names. If someone wants to write a small script to do that, I think this source can be kept rather up-to-date.

Currently on Dropbox since S3 refused a 160MB file. If you could lift the limits, can upload there. Otherwise this could be a temporary solution.

### CODE REVIEW

**Comment by sergiyprotsiv** [comment]
If successful, this also supersedes at least my older PR: https://github.com/openaddresses/openaddresses/pull/2240
And possibly others, like https://github.com/openaddresses/openaddresses/pull/2275. 

It has to be noted though that this dataset does not have the postal code, so maybe others are still relevant.

**Comment by iandees** [comment]
Wooo! Look at all those dots!

**Comment by migurski** [comment]
I’ve raised the size limit to 256MB so it should now be possible to upload this file.

File: sources/de/nw/statewide.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+	"coverage": {
+        "country": "de",
+        "state": "nw"
+    },
```
Comment (migurski) [inline sources/de/nw/statewide.json]: Can we add an ISO code here? Should look like this [based on the coverage docs](https://github.com/openaddresses/openaddresses/blob/master/CONTRIBUTING.md#coverage-object):

```JSON
"ISO 3166": { "alpha2": "DE-NW" }
```

**Pull request merged**